edit-post/store/index.js

```diff
@@ -1,24 +1,39 @@
 /**
  * WordPress Dependencies
  */
-import { registerReducer, withRehydratation, loadAndPersist } from '@wordpress/data';
+import {
+	registerStore,
+	withRehydratation,
+	loadAndPersist,
+	subscribe,
+	dispatch,
+} from '@wordpress/data';
 
 /**
  * Internal dependencies
  */
 import reducer from './reducer';
-import enhanceWithBrowserSize from './mobile';
-import { BREAK_MEDIUM } from './constants';
+import * as actions from './actions';
+import * as selectors from './selectors';
 
 /**
  * Module Constants
  */
 const STORAGE_KEY = `WP_EDIT_POST_PREFERENCES_${ window.userSettings.uid }`;
-const MODULE_KEY = 'core/edit-post';
 
-const store = registerReducer( MODULE_KEY, withRehydratation( reducer, 'preferences', STORAGE_KEY ) );
+const store = registerStore( 'core/edit-post', {
+	reducer: withRehydratation( reducer, 'preferences', STORAGE_KEY ),
+	actions,
+	selectors,
+} );
 
 loadAndPersist( store, reducer, 'preferences', STORAGE_KEY );
-enhanceWithBrowserSize( store, BREAK_MEDIUM );
+
+subscribe( 'core/viewport', [ 'isViewportMatch', '< medium' ], ( isSmall ) => {
+	// Collapse sidebar when viewport shrinks.
+	if ( isSmall ) {
+		dispatch( 'core/edit-post' ).closeGeneralSidebar();
```

Comment:
I believe this is a new behavior, how was this handled before? The separation between the mobile sidebar and the desktop sidebar?

Reply:
It was previously handled in `edit-post/store/reducer.js`'s handling of the `UPDATE_MOBILE_STATE` action triggered by `enhanceWithBrowserSize`.